pints/_mcmc/__init__.py

```diff
@@ -274,7 +274,9 @@ class MCMCController(object):
     ----------
     log_pdf : pints.LogPDF
         A :class:`LogPDF` function that evaluates points in the parameter
-        space.
+        space, or a list of :class:`LogPDF` of the same length as `chains`. If
+        multiple LogPDFs are provided, each chain will call only its
+        corresponding LogPDF.
```

Comment:
Needs a note either here or in set_parallel to say we can't have parallel execution of multiple logpdfs